programmers-coding-test/minjae/greedy/그리디3-숫자-카드-게임.js

```diff
@@ -0,0 +1,27 @@
+function solution(row, column) {
+    const RandomArrays = [];
+    const minNumbers= [];
+    
+    let answer = 0;
+
+    for (let i = 0; i < row; i++) {
+
+        let temp = [];
+        for (let j = 0; j < column; j++) {
+            temp.push(Math.floor(Math.random() * 9 + 1));
+        }
+        RandomArrays.push(temp);
+    }
+
+    for (let i = 0; i < RandomArrays.length; i++) {
+        minNumbers.push(Math.min.apply(null, RandomArrays[i]));
+    }
+
+    /* 출력 확인용 */
+    console.log(RandomArrays);
+    console.log(minNumbers);
+
+    return answer = Math.max.apply(null, minNumbers);
```

Comment:
Iterable 객체의 map 메서드를 이용하면 한 줄로 해결할 수 있다.
```suggestion
  return Math.max( ...RandomArrays.map(cards => Math.min(...cards)) );
```